src/brevitas/graph/calibrate.py

```diff
@@ -57,9 +57,28 @@ def __enter__(self):
             self.disable_quant_inference.apply(self.model, is_training=True, quantization_enabled=False)
 
     def __exit__(self, type, value, traceback):
+        self.model.apply(finalize_collect_stats)
```

Comment:
This one and related should be a separate PR, doesn't have anything to do with bias correction. The tutorial should also be updated.